html/images/svg/crowdsourcing-obf-round-smartphones.svg

```diff
@@ -0,0 +1,432 @@
+     inkscape:pageopacity="0"
+     guidetolerance="10"
+     gridtolerance="10"
+     objecttolerance="10"
+     borderopacity="1"
+     bordercolor="#666666"
+     pagecolor="#ffffff" />
+  <!-- Generator: Sketch 63.1 (92452) - https://sketch.com -->
+  <title
+     id="title2">Artboard</title>
+  <desc
+     id="desc4">Created with Sketch.</desc>
+  <defs
+     id="defs8">
+    <polygon
+       points="1.87540449e-13 -9.38407461e-14 47.4266681 -9.38407461e-14 47.4266681 63.9709735 1.87540449e-13 63.9709735"
+       id="path-1" />
+    <polygon
+       points="3.72691847e-13 -1.86454816e-13 94.2491747 -1.86454816e-13 94.2491747 127.105725 3.72691848e-13 127.105725"
+       id="path-3" />
+  </defs>
+  <path
+     d="M 293,534.49768 C 137.21961,558.14716 121.5131,409.89704 99,270"
+     id="path8563"
+     stroke="#000000"
+     stroke-width="18.8117"
+     style="fill:none;fill-rule:evenodd" />
+  <path
+     d="M 589,532.50232 C 744.78036,508.85284 760.48688,657.10295 783,797"
+     id="path8599"
+     stroke="#000000"
+     stroke-width="18.8117"
+     style="fill:none;fill-rule:evenodd" />
+  <rect
+     id="rect6788"
+     fill="#000000"
+     x="296"
+     y="777"
+     width="309"
+     height="132"
+     style="fill-rule:evenodd;stroke:none;stroke-width:1" />
+  <path
+     d="M 437,455 C 310.13181,455 208,557.10805 208,683.94673 V 908 H 666 V 683.94673 C 666,557.10805 563.86819,455 437,455 Z"
+     id="rect8544"
+     fill="#0066ff"
+     style="fill-rule:evenodd;stroke:none;stroke-width:1" />
+  <rect
+     ry="20.266657"
+     id="rect4493"
+     fill="#1a1a1a"
+     transform="rotate(7.53048)"
+     x="63.741817"
+     y="10.739026"
+     width="128.27856"
+     height="237.4518"
+     style="fill-rule:evenodd;stroke:none;stroke-width:1" />
+  <rect
+     id="rect4495"
+     fill="#cccccc"
+     transform="rotate(7.53048)"
+     x="73.97699"
+     y="32.573639"
+     width="107.80858"
+     height="193.7825"
+     style="fill-rule:evenodd;stroke:none;stroke-width:1" />
+  <circle
+     id="path8542"
+     fill="#ffcc00"
+     cx="437"
+     cy="293"
+     r="174"
+     style="fill-rule:evenodd;stroke:none;stroke-width:1" />
+  <g
+     id="Group-2"
+     transform="rotate(-39,172.05935,60.929411)"
+     style="fill:none;fill-rule:evenodd;stroke:none;stroke-width:1">
+    <polygon
+       id="Path"
+       fill-opacity="0"
+       fill="#000000"
+       points="108.64571,78.836588 78.086356,42.079345 127.52696,0.83658763 158.08636,37.59383 " />
+    <g
+       id="Group"
+       transform="rotate(50.213008,62.344851,136.41917)">
+      <g
+         id="Clipped"
+         transform="translate(-5.2e-5,0.001066)">
+        <mask
+           id="mask-2"
+           fill="#ffffff">
+          <use
+             xlink:href="#path-1"
+             id="use18"
+             x="0"
+             y="0"
+             width="100%"
+             height="100%" />
+        </mask>
+        <g
+           id="g21" />
+        <image
```

Comment:
That's not good, we shouldn't have a bitmap image in a SVG...
That's the bottle cap, you can find a vector version of it in the OFF logo.